tests/python/unittest/test_ndarray.py

```diff
@@ -590,9 +590,8 @@ def gt_topk(dat, axis, ret_typ, k, is_ascend):
     gt = gt_topk(a_npy, axis=None, ret_typ="indices", k=5*5*5*5, is_ascend=False)
     assert_almost_equal(nd_ret_argsort, gt)
 
-    # test topk with a big shape
-    a = mx.nd.arange(0, 54686454, step=1, repeat=1)
-    assert_almost_equal(a.topk(k=54686454).asnumpy(), a.asnumpy()[::-1])
+    a = mx.nd.arange(0, 1024, step=1, repeat=1)
+    assert_almost_equal(a.topk(k=1024).asnumpy(), a.asnumpy()[::-1])
```

Comment:
`k = 1024` cannot test against the failure in https://github.com/apache/incubator-mxnet/issues/8303. What platforms does this test crash?

By the way, here is another test for big shape, https://github.com/apache/incubator-mxnet/blob/master/tests/python/unittest/test_optimizer.py#L237

Reply:
Raspberry PI, has only 1G of memory + swap. We are doing something wrong if we have to write a test with such a big allocation, if the type is wrong as in the issue that we have linked, this should be mocked, or tested in a smarted way. Not just allocating huge arrays willy-nilly.

Reply:
Agree that it's a bit heavy for an unit test. Shall we move it to the nightly test suite?